src/cdk-experimental/popover-edit/constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/** Selector for finding table cells. */
+export const CELL_SELECTOR = '.cdk-cell, .mat-cell, td';
```

Comment:
Longer term, the thing to do is probably to make that selector customizable via some DI config.

I included mat in there on the off chance someone would want to have a more stylistically customizable edit for their mat table than what MatPopoverEdit will be.

Reply:
Selectors have to be completely static because they're resolved at compile time into generated code. We'd have to do something like creating a child class.

Reply:
Is that true for this case where it's applying the selector imperatively?

Reply:
Nvm, I didn't follow-through on how it was being used.